dbt/adapters/databricks/python_submissions.py

```diff
@@ -18,6 +18,8 @@ def __init__(self, parsed_model: Dict, credentials: DatabricksCredentials) -> No
             parsed_model=parsed_model, credentials=credentials  # type: ignore[arg-type]
         )
 
+        self.database = parsed_model.get("database")
```

Comment:
Just to confirm: if a user specifies `catalog`, it will still be stored in this database entry right?

Reply:
Yes, it will be stored here.